static/app/views/issueDetails/traceTimeline/traceTimelineTooltip.tsx

```diff
@@ -0,0 +1,115 @@
+import {Link} from 'react-router';
```

Comment:
Think this should be the sentry link component instead (I auto-import the wrong one all the time)